### PR TITLE
Adds Envoy 1.19.0

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Ran across this, so figured I would update it if you like.

https://archive.tetratelabs.io/envoy/envoy-versions.json

### CODE REVIEW

File: envoy.hcl [hunk 1]
```diff
@@ -3,7 +3,7 @@ test = "envoy --version"
 strip = 2
 binaries = ["envoy"]
 
-// Artefacts are from the Tetrate Labs manifest:
+// Artifacts are from the Tetrate Labs manifest:
```
Comment (alecthomas) [inline envoy.hcl:6]: Believe it or not, "artefacts" is how it is spelt in Australia. Please revert.

**Comment by alecthomas** [comment]
Thanks for the PR, just one minor comment.

File: envoy.hcl (hunk 1 above)
Comment (codefromthecrypt) [reply envoy.hcl:6]: sure thing. odd I never learned this, yet!

```suggestion
// Artefacts are from the Tetrate Labs manifest:
```

**Comment by alecthomas** [comment]
Awesome, thanks!

**Pull request merged**